RedditSharp/Things/Post.cs

```diff
@@ -109,7 +114,7 @@ public Comment[] Comments
         public Uri Permalink { get; set; }
 
         [JsonProperty("score")]
-        public int Score { get; set; }
+        public new int Score { get; set; }
```

Comment:
Instead of new, just remove this entire property because there is no need to block the inheritance on it